routes/index.js

```diff
@@ -1,6 +1,6 @@
-var express = require('express')
-var router = express.Router()
-var db = require('../db/index.js')
+const express = require('express')
+const router = express.Router()
+const db = require('../db')
```

Comment:
[nit] Does /index.js not need to be specified? There are currently 2 files in db folder.

Reply:
I think this kinda explains it? https://www.reddit.com/r/javascript/comments/7rv221/to_indexjs_or_not_to_indexjs/